bin/pycbc_multi_inspiral

```diff
@@ -661,9 +665,11 @@ with ctx:
                         bank_chisq.values(template, stilde[ifo].psd, stilde[ifo],
                                          coherent_ifo_triggers[ifo] /
                                          norm_dict[ifo], norm_dict[ifo],
-                                         coinc_idx+stilde[ifo].analyze.start)
+                                         coinc_idx_det_frame[ifo]+stilde[ifo].analyze.start)
```

Comment:
Where possible, try to keep line length to 80 characters (it's not always possible to do this)